frontend/src/components/Review/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactElement } from 'react';
+import styles from './Review.module.scss';
+import Card from 'react-bootstrap/Card';
+import Rating from './Rating';
+
+type Props = {
+  readonly overall_rating: number;
+  readonly date: Date;
+  readonly text: string;
+};
+
+const months = [
+  'Jan',
+  'Feb',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'Aug',
+  'Sept',
+  'Oct',
+  'Nov',
+  'Dec',
+];
+
+const date_to_string = (date: Date): string => {
```

Comment:
better to refactor this into a util script outside of the components folder so that we can reuse this if needed.